velox/functions/sparksql/tests/ArrayInsertTest.cpp

```diff
@@ -0,0 +1,329 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/functions/sparksql/tests/SparkFunctionBaseTest.h"
+
+using namespace facebook::velox::test;
+
+namespace facebook::velox::functions::sparksql::test {
+namespace {
+
+class ArrayInsertTest : public SparkFunctionBaseTest {
+ protected:
+  void SetUp() override {
+    SparkFunctionBaseTest::SetUp();
+    options_.parseIntegerAsBigint = false;
+  }
+
+  void testExpression(
+      const std::string& expression,
+      const std::vector<VectorPtr>& input,
+      const VectorPtr& expected) {
+    const auto result = evaluate(expression, makeRowVector(input));
+    assertEqualVectors(expected, result);
+  }
+
+  void testExpression(
+      const std::string& expression,
+      const RowVectorPtr& input,
+      const VectorPtr& expected) {
```

Comment:
This API looks similar with that at L30. Wondering if one if enough.

Reply:
> but for most of the tests need to wrap a RowVectorPtr for the input vectors which is kind of duplicated code.

If that's the case, to avoid duplicate wrapping for RowVectorPtr, can we remove the API at L30 and use this one for all tests?

Reply:
You are right. Removed the other function. Thanks.